### PR TITLE
use more consistent "snap %q (rev %s)" task descriptions

### PR DESCRIPTION
**Pull request by mvo5**
As discussed in the enable/disable PR this branch moves towards more consistent task descriptions. The form is `... snap %s (rev %s) ...`. Happy to tweak as needed of course.

### CODE REVIEW

**Comment by niemeyer** [comment]
Suspect the test breakages are real, side effect of greps for task summaries.

File: overlord/snapstate/snapstate.go [hunk 1]
```diff
@@ -269,10 +269,10 @@ func Enable(s *state.State, name string) (*state.TaskSet, error) {
 		Revision: snapst.Current,
 	}
 
-	prepareSnap := s.NewTask("prepare-snap", fmt.Sprintf(i18n.G("Prepare snap %q (%s)"), ss.Name, snapst.Current))
+	prepareSnap := s.NewTask("prepare-snap", fmt.Sprintf(i18n.G("Prepare snap %q (rev %s)"), ss.Name, snapst.Current))
```
Comment (niemeyer) [inline overlord/snapstate/snapstate.go]: Oops.. %q (%s)..

File: overlord/snapstate/snapstate.go [hunk 2]
```diff
@@ -269,10 +269,10 @@ func Enable(s *state.State, name string) (*state.TaskSet, error) {
...
 	prepareSnap.Set("snap-setup", &ss)
 
-	linkSnap := s.NewTask("link-snap", fmt.Sprintf(i18n.G("Make snap %q (%s) unavailable to the system%s"), ss.Name, snapst.Current))
+	linkSnap := s.NewTask("link-snap", fmt.Sprintf(i18n.G("Make snap %q (rev %s) available to the system%s"), ss.Name, snapst.Current))
```
Comment (niemeyer) [inline overlord/snapstate/snapstate.go]: Same

File: overlord/snapstate/snapmgr_test.go [hunk 3]
```diff
@@ -486,7 +486,7 @@ func (s *snapmgrTestSuite) TestInstallRunThrough(c *C) {
 
 	// check link snap summary
 	linkTask := ta[len(ta)-1]
-	c.Check(linkTask.Summary(), Equals, `Make snap "some-snap" available to the system (rev 11)`)
+	c.Check(linkTask.Summary(), Equals, `Make snap "some-snap" (rev 11) available to the system`)
```
Comment (niemeyer) [inline overlord/snapstate/snapmgr_test.go]: Sorry, I thought it was only on the other file that it was off, but it seems to use rev consistently.

It seems slightly nicer to standardize as "... snap %q (%s) ..." as we discussed online. It looks slightly nicer, it's shorter which is good as we'll use it a lot, and people will very quickly learn that this is the revision.

**Comment by niemeyer** [comment]
LGTM, one suggestion about the format, and tests need fixing.

**Comment by mvo5** [comment]
retest this please

**Pull request merged**